test/support/codeflare.go

```diff
@@ -50,6 +60,28 @@ func GetPyTorchImage() string {
 	return lookupEnvOrDefault(CodeFlareTestPyTorchImage, "pytorch/pytorch:1.11.0-cuda11.3-cudnn8-runtime")
 }
 
+func GetInstaScaleOcmSecretName() string {
+	return lookupEnvOrDefault(InstaScaleOcmSecretName, "instascale-ocm-secret")
+}
+
+func GetInstaScaleOcmSecretNamespace() string {
+	return lookupEnvOrDefault(InstaScaleOcmSecretNamespace, "default")
+}
+
+func GetOsdClusterId() (string, bool) {
+	return os.LookupEnv(OsdClusterID)
+}
+
+func IsOsd() bool {
+	env := lookupEnvOrDefault(IsOSD, "false")
+	osd, err := strconv.ParseBool(env)
+	if err != nil {
+		fmt.Errorf("error parsing IS_OSD variable, error: %v ", err)
```

Comment:
Maybe would be better to use `test.Expect(err).NotTo(gomega.HaveOccurred(), fmt.Sprintf("error parsing IS_OSD variable, error: %v ", err))` here. That will make sure the error is propagated to the output.